http.go

```diff
@@ -628,7 +628,7 @@ func (c *Client) Nodes(ctx context.Context) (json.RawMessage, error) {
 
 // Ready returns the readiness of the node. The caller must close the returned ReadCloser
```

Comment:
It's actually not a bug, as you can see from the comment. The `/ready` endpoint doesn't return JSON response.

https://rqlite.io/docs/guides/monitoring-rqlite/#readiness-checks

I'm open to suggestions for a better return type, but this is not it.